src/index.js

```diff
@@ -2,9 +2,9 @@ import React from 'react'
 import invariant from 'invariant'
 import * as Utils from './utils'
 
-export { default as useUncontrolled } from './hook.js';
+export { default as useUncontrolled } from './hook.js'
 
-export function uncontrollable(
+export default function uncontrollable(
```

Comment:
Mixing default and named exports may lead to unexpected bugs for dependent packages without esm support (like react-overlays I guess). I already shout in my foot in material-ui with this.

Reply:
b/c the compiled output is different for the `require` case you mean?

Reply:
Yes, webpack is buggy here. The result is compiled into
```js
module.exports = {
  default: {
    default: function () {}
  }
}
```
So babel interop is not able to solve this.

Reply:
i just added different index's for each case that should handle it I believe